stack.yaml

```diff
@@ -9,7 +9,6 @@ resolver: lts-11.6
 #       name: "fpco/stack-test"
 nix:
   # --nix on the command-line to enable.
-  enable: false
```

Comment:
What's the motivation for this change?

Reply:
Before, `enable: false` was the default, so this line had no effect. After this PR is applied, the default value depends on whether we're running under NixOS or not, so it's desirable to omit it to build correctly on both NixOS and other distros.

Reply:
Makes sense, thanks!